docs/content/docs/data-structures/tree.md

```diff
@@ -270,10 +267,21 @@ const containerSchema: ContainerSchema = {
 const { container, services } = await client.createContainer(containerSchema);
 ```
 
-Apply the schema to the tree by passing the tree configuration object to the `ITree.schematize()` method.
+Use `ITree.viewWith` to create a `TreeView` based on your tree configuration.
+Tree views provide schema-dependent APIs for viewing and editing tree data.
 
 ```typescript
-const stickyNotesTreeView = container.initialObjects.appData.schematize(appTreeConfiguration);
+const stickyNotesTreeView = container.initialObjects.appData.viewWith(appTreeConfiguration);
+```
+
+When the tree is first created, this schema along with some initial data can be applied to the tree using `TreeView.initialize`.
+Note that the parameter to initialize must conform to the root schema, `App`. So in this example, it has a single `items` property. The value of the `items` property specifies that the items array is empty. It is not a requirement that the initial tree be empty: you can assign one or more groups or notes to the initial tree.
```

Comment:
the "parameter to initialize" wording seems kinda off to me, maybe something more like "data to initialize with"?